client/src/js/todos/addCard.js

```diff
@@ -1,73 +1,114 @@
-  };
+  }
 
-  postCardHandler = (target) => {
+  postCard(target) {
     const content = $('.item-content', closest('.list_item', target));
     const maxLength = 200;
-    content.value.length > maxLength
-      ? alert('글자수를 초과하였습니다.')
-      : this.renderPostedCard(target);
-  };
+    if (content.value.length > maxLength) {
+      alert('글자수를 초과하였습니다.');
+    } else {
+      const cardItem = closest('.list_item', target);
+      const title = $('.item-title', cardItem).value;
+      const content = $('.item-content', cardItem).value;
+      const state = $('.column-title', closest('.column', target)).textContent;
+      const cardId = cardItem.dataset.id;
+      this.postCardData(title, content, state, cardId);
+      this.updateSequence(cardItem);
+      this.renderPostedCard(title, content, cardItem);
+    }
+  }
 
-  inputFocusHandler = (e) => {
+  inputFocusHandler(e) {
     const accentBtn = $('.accent-btn', closest('.list_item', e.target));
-    const thisItem = closest('.list_item', e.target);
-    const title = $('.item-title', thisItem).value;
-    const content = $('.item-content', thisItem).value;
+    const cardItem = closest('.list_item', e.target);
+    const title = $('.item-title', cardItem).value;
+    const content = $('.item-content', cardItem).value;
     title && content
       ? accentBtn.removeAttribute('disabled')
       : (accentBtn.disabled = 'true');
-  };
-
-  renderPostedCard = (target) => {
-    const thisItem = closest('.list_item', target);
-    const title = $('.item-title', thisItem).value;
-    const content = $('.item-content', thisItem).value;
-    thisItem.classList.remove('creating');
-    thisItem.classList.replace('active', 'default');
-    thisItem.innerHTML = this.postCardTemplate({
-      title,
-      content,
-    });
-  };
-
-  addCardEventHandler = (e) => {
-    if (closest('.plus-btn', e.target)) {
-      this.addCardHandler(e);
-    }
+  }
 
-    if (containClass(e.target, 'cancel-add-btn')) {
-      this.cancelAddHandler(e);
-    }
+  renderPostedCard(title, content, cardItem) {
+    cardItem.classList.remove('creating');
+    cardItem.classList.replace('active', 'default');
+    cardItem.innerHTML = this.postCardTemplate(title, content);
+  }
 
-    if (containClass(e.target, 'post-btn')) {
-      e.preventDefault();
-      this.postCardHandler(e.target);
-    }
-  };
+  postCardData(todoTitle, todoContent, state, cardId) {
+    const url = getURL('cards');
+    const data = {
+      id: cardId,
+      title: todoTitle,
+      content: todoContent,
+      lastTime: new Date(),
+      states: state,
+    };
+
+    requestToServer(url, 'POST', data);
+  }
+
+  async updateSequence(cardItem) {
+    const columnName = $(
+      '.column-title',
+      closest('.column', cardItem)
+    ).textContent;
+
+    const sequence = await getSequenceData(columnName);
+    sequence.unshift(Number(cardItem.dataset.id));
+
+    const patchData = {};
+    patchData[columnName] = sequence;
```

Comment:
const로 작성해도 객체는 주소로 값을 참조하고 있어서 변경이 가능하긴 한데요.. 음 제 생각에는 최대한 const를 사용할 때에는 값을 참조해서 변경하지 않는 편이 좋아 보입니다!  
`const patchData = {a: 1}`로 만들었는데 다른 메소드로 넘어가서 값이 변경된다면 변경부를 알아내는데 어려움이 있을 수 있어서 그렇습니다! 
아래 방식으로도 객체를 만들어 줄 수 있으니 참고 부탁드립니다~ 
```suggestion
    const patchData = { [columnName]: sequence };
```

<img width="415" alt="Screen Shot 2022-04-16 at 10 05 10" src="https://user-images.githubusercontent.com/30427711/163655859-8eb31288-f0df-4511-b944-4b0240aaccd4.png">